src/GithubLine/index.js

```diff
@@ -13,7 +13,7 @@ export default class GithubLine extends Component {
     count: `–––`
   }
 
-  componentWillMount() {
+  UNSAFE_componentWillMount() {
```

Comment:
let's create a ticket to migrate this?

Reply:
https://github.com/iterative/dvc.org/issues/584